### PR TITLE
Refactor Help Component

### PR DESCRIPTION
**Pull request by commy2**
**When merged this pull request will:**
- streamlining / modernizing some code
- removes random component line from main menu and pause menu
- insteads shows full list in diary

### CODE REVIEW

**Comment by dedmen** [comment]
But I liked the random component line ._.
Who looks into diary, like ever?

**Comment by commy2** [comment]
I restored some commented out diary entries (with updated links).
<details><summary>Wiki</summary>



</details>

<details><summary>Bugtracker</summary>



</details>

<details><summary>Credits</summary>



</details>

Credits is basically the same as the main menu random component line and I like the full list (that is less in your face) more.

I think Bugtracker and Wiki are superfluous. And you can't have clickable links in the diary. Should they be removed entirely?

Keybinding is the same, except that I deleted a few newlines at the end of the entry, so the spacing looks prettier.

**Comment by jonpas** [comment]
I thought diary supports HTML including links?

Bug tracker and wiki could stay, but maybe under a single header, not two.

**Comment by commy2** [comment]
It's fake HTML with different tags:
https://community.bistudio.com/wiki/createDiaryRecord

**Comment by commy2** [comment]
<details><summary>ACE and CBA</summary>





</details>

Note that `ace - etc.` will be displayed as `@ace` for most people, since it is the folder name of the addon.

File: addons/help/XEH_postInit.sqf [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        } forEach _addonActions;
+
+        _keys = _keys + "<br/>";
+    } forEach _addons;
+
+    // delete last line breaks
+    _keys = _keys select [0, count _keys - 10];
+
+    _unit createDiaryRecord [QGVAR(docs), [localize "STR_CBA_Help_Keys", _keys]];
+    _unit createDiaryRecord [QGVAR(docs), [localize "STR_CBA_Credits", call (uiNamespace getVariable QGVAR(credits))]];
```
Comment (PabstMirror) [inline addons/help/XEH_postInit.sqf]: I think this is failing with large modpack (`count call (uiNamespace getVariable "cba_help_credits")` = 33772)
rpt says ` Unknown entity: ' Accessories</font> by Red Hammer Studios</font><br/><font colo`
that part is exactly 3350 chars into the `call (uiName` string

Comment (commy2) [reply addons/help/XEH_postInit.sqf]: `createDiaryRecord` has a hard limit for chars?

Comment (commy2) [reply addons/help/XEH_postInit.sqf]: It was the & conflicting with the pseudo html of the diary / sqf TEXT.

**Pull request merged**